errno/errname.go

```diff
@@ -868,6 +869,7 @@ var MySQLErrName = map[uint16]string{
 	ErrRoleNotGranted:                                        "%s is is not granted to %s",
 	ErrMaxExecTimeExceeded:                                   "Query execution was interrupted, max_execution_time exceeded.",
 	ErrLockAcquireFailAndNoWaitSet:                           "Statement aborted because lock(s) could not be acquired immediately and NOWAIT is set.",
+	ErrNotHintUpdatableVariable:                              "Variable '%s' cannot be set using %s hint.",
```

Comment:
From MySQL document https://dev.mysql.com/doc/mysql-errors/8.0/en/server-error-reference.html, the error pattern should be:

```suggestion
	ErrNotHintUpdatableVariable:                              "Variable %s cannot be set using SET_VAR hint.",
```

Reply:
I tested it with the local MySQL, and the result returned as `Variable '%s' cannot be set using SET_VAR hint.`
The version of MySQL is 8.0.21

Reply:
confirmed:

```sql
MySQL(root@127.0.0.1:test) > select /*+ set_var(innodb_api_enable_binlog=1) */ * from t;
Empty set, 1 warning (0.00 sec)

MySQL(root@127.0.0.1:test) > show warnings;
+---------+------+-----------------------------------------------------------------------+
| Level   | Code | Message                                                               |
+---------+------+-----------------------------------------------------------------------+
| Warning | 3637 | Variable 'innodb_api_enable_binlog' cannot be set using SET_VAR hint. |
+---------+------+-----------------------------------------------------------------------+
1 row in set (0.02 sec)
```